pkgs/tools/networking/rpki-arin-tal/default.nix

```diff
@@ -0,0 +1,31 @@
+{ lib, stdenv, fetchurl, writeScript, bash }:
+
+stdenv.mkDerivation rec {
+  pname = "rpki-arin-tal";
+  version = "1.0";
+
+  src = fetchurl {
+    url = "https://www.arin.net/resources/manage/rpki/arin.tal";
```

Comment:
```suggestion
    url = "https://web.archive.org/web/20221023232257/https://www.arin.net/resources/manage/rpki/arin.tal";
```

Reply:
Too lazy to spell out why you see? The url works for me.

Reply:
The URL is unversioned and could break over time.